cves/kernel/CVE-2016-8650.yml

```diff
@@ -356,8 +373,11 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
```

Comment:
Add a period at the end. Consider separating this statement into two sentences for improved readability. You could replace your second instance of the word "to" with the word "and".